core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -653,7 +653,11 @@ public void addColumn(
         );
 
         if (!Os.isWindows()) {
-            ff.fsyncAndClose(openRO(ff, path.$(), LOG));
+            try {
+                ff.fsyncAndClose(openRO(ff, path.$(), LOG));
+            } catch (CairoException e) {
+                LOG.info().$("could not fsync after column added, non-critical [path=").$(path).$();
```

Comment:
this should be error, perhaps critical too, fsync have to be investigated

Reply:
ok, done